cocos2d/actions/tween.js

```diff
@@ -215,9 +215,9 @@ Tween.stopAllByTarget = function (target) {
  * Insert an action or tween to this sequence
  * !#zh
  * 插入一个 action 或者 tween 到队列中
- * @method then 
- * @param {Action|Tween} other
- * @return {Tween}
+ * @method then
+ * @param {Action|Tween<T>} other
+ * @return {Tween<T>}
```

Comment:
I don't think document-generator can recognize genericity in `param` and `return`. You need to add a decorator 'typescript' likes below
```
@typescript then (othen: Action|Tween<T>): Tween<T>
```

Reply:
I have tested it with the following tools, it works:
https://github.com/cocos-creator/creator-api-docs and
https://github.com/cocos-creator/firedoc

It is an internal workflow, so you can decide which one is better.